src/store/flow/flow.ts

```diff
@@ -361,14 +361,15 @@ export const actions: ActionTree<IFlowsState, IRootState> = {
 
     // Set UI positions
     // TODO: remove this once IBlock has vendor_metadata key
```

Comment:
@bzabos , I think we missed to remove this TODO, can we take the opportunity to remove this ?